conan/tools/cmake/toolchain/blocks.py

```diff
@@ -273,17 +273,33 @@ class CppStdBlock(Block):
     template = textwrap.dedent("""\
         # Define the C++ and C standards from 'compiler.cppstd' and 'compiler.cstd'
 
+        function(conan_modify_std_watch variable access value current_list_file original_value)
```

Comment:
Maybe we can just have 1 function instead of 3, and this function just checks ``variable`` name to compare against ``cstd`` or ``cppstd``?

Reply:
Good call, I think the resulting diff is cleaner